src/tools/cephfs/cephfs-shell

```diff
@@ -814,28 +816,26 @@ sub-directories, files')
                 dirs = []
                 for i in all_items:
                     for item in ls(path):
-                        d_name = item.d_name
+                        d_name = item.d_name.decode('utf-8')
```

Comment:
We should not convert any paths to strings, at least in new code. The better approach here would be convert every path in `args.paths` to a bytes array. Write a helper method to do that. How about a separate PR that fixes all the paths?

Reply:
Okay. I will create a new PR for it.